src/Microsoft.DocAsCode.Build.Engine/PostbuildPhaseHandler.cs

```diff
@@ -9,11 +9,14 @@ namespace Microsoft.DocAsCode.Build.Engine
 
     internal class PostbuildPhaseHandler : IPhaseHandler
     {
-        private DocumentBuildContext _context;
+        public DocumentBuildContext Context { get; }
```

Comment:
>Context [](start = 36, length = 7)

some Handlers have public Context while some have private context, what is the rule?